app/views/MessagesView/index.tsx

```diff
@@ -73,17 +102,17 @@ class MessagesView extends React.Component {
 		});
 	};
 
-	navToRoomInfo = navParam => {
+	navToRoomInfo = (navParam: { rid: string }) => {
 		const { navigation, user } = this.props;
 		if (navParam.rid === user.id) {
 			return;
 		}
 		navigation.navigate('RoomInfoView', navParam);
 	};
 
-	jumpToMessage = async ({ item }) => {
+	jumpToMessage = async ({ item }: { item: IMessageItem }) => {
 		const { navigation, isMasterDetail } = this.props;
-		let params = {
+		let params: any = {
```

Comment:
Can't we create an `IParams` interface for this? It seems it has values by default so we are able to know which values it receives.